scripts/diff-release-package.mts

```diff
@@ -0,0 +1,123 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.dev/license
+ */
+
+/**
+ * Script that can be used to compare the local `npm_package` snapshot artifact
+ * with the snapshot artifact from GitHub at upstream `HEAD`.
+ *
+ * This is useful during the `rules_js` migration to verify the npm artifact
+ * doesn't differ unexpectedly.
+ *
+ * Run via:
+ *    node --loader ts-node/esm/transpile-only scripts/diff-release-package.mts <pkg-name>
+ */
+
+import { GitClient } from '@angular/ng-dev';
+import childProcess from 'node:child_process';
+import fs from 'node:fs';
+import os from 'node:os';
+import path from 'node:path';
+import sh from 'shelljs';
+
+// Do not remove `.git` as we use Git for comparisons later.
+// Also preserve `uniqueId` as it's irrelevant for the diff and not included via Bazel.
+// The `README.md` is also put together outside of Bazel, so ignore it too.
+const SKIP_FILES = ['README.md', 'uniqueId', '.git'];
+
+const packageName = process.argv[2];
+if (!packageName) {
+  console.error('Expected package name to be specified.');
+  process.exit(1);
+}
+
+try {
+  await main(packageName);
+} catch (e) {
+  console.error(e);
+  process.exitCode = 1;
+}
+
+async function main(packageName: string) {
```

Comment:
If desired, you can leverage the runner we already use for scripts in the Angular CLI, eliminating the need to invoke the function directly, as shown above.

To execute the script, use the following command:

```bash
yarn admin diff-release-package @angular/build
```

For instance, the updated script could look like this:

```typescript
export default async function (argv: { _: string[] }) {
  const packageName = argv._[0];
```

Reply:
Replied above regarding the script runner.